website/management/commands/check_trademarks.py

```
@@ -12,27 +12,53 @@
+
+        # If there is no scroll_id, it's possible there are no results or they are in the first response
+        if not scroll_id:
+            results = response_json.get("results")
+            return {"count": len(results) if results else 0}
+
+        pagination_payload = {
+            "keywords": f'["{term}"]',
+            "start_index": "0",
+            "scroll_id": scroll_id,
+        }
+        response = requests.post(url, data=pagination_payload, headers=headers)
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)